### ISSUE TITLE
Why does PSDepend Bootstrap-Nuget even if we're not using nuget dependencies

### ISSUE THREAD

**Issue by gaelcolas** [RamblingCookieMonster/PSDepend#117]
I found that it [always tries to download Nuget](https://github.com/RamblingCookieMonster/PSDepend/blob/master/PSDepend/PSDepend.psm1#L31) from the web even when Nuget is not used as a dependency... (i.e. when using PS Module dependencies).

I also don't see a way to configure PSDepend "not to do that".

Is there any reason I'm missing, or would you accept a PR to change that (ideally, a param would be good, but because it's at module load, I wonder how we could make this work?

**Comment by RamblingCookieMonster** [comment]
Yeah, totally should not do that if we can avoid it!  Open to suggestions.  Including it in the nuget dependency type?  Environment variable (:shrug:)?

**Comment by gaelcolas** [comment]
thanks for sanity check :)

I'll have a closer look at the code, but I'm not a fan of environment variables for that.

In the meantime, my hack is: `Set-Alias 'nuget.exe' echo` 🤣

**Comment by HeyItsGilbert** [comment]
We should definitely address this.
